lib/models/wallet_connect/approve_token_transfer_result.dart

```diff
@@ -0,0 +1,16 @@
+/// Results for approving token transfer
+class ApproveTokenTransferResult {
+  final int? tick;
+  final String? errorMessage;
+  final int? errorCode;
+  ApproveTokenTransferResult(
+      {required this.tick, this.errorCode, this.errorMessage});
+
```

Comment:
@Qubic-Hub  We received the following feedback: 
In order for walletconnect clients to track transaction status qubic_sendQubic should include either only the transaction ID, or the signed transaction bytes.
Can we include this here and in all the methods that are generating and signing a transaction?
https://github.com/qubic/wallet-app-dapp/issues/2